ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -366,6 +366,8 @@ objects:
           queue /tmp/artifacts/configmaps.json oc --insecure-skip-tls-verify --request-timeout=5s get configmaps --all-namespaces -o json
           queue /tmp/artifacts/csr.json oc --insecure-skip-tls-verify --request-timeout=5s get csr -o json
           queue /tmp/artifacts/endpoints.json oc --insecure-skip-tls-verify --request-timeout=5s get endpoints --all-namespaces -o json
+          queue /tmp/artifacts/deployments.json oc --insecure-skip-tls-verify --request-timeout=5s get deployments --all-namespaces -o json
```

Comment:
what use-case is this fulfilling?

Reply:
Gathering information about the state of operands after a run fails.  Notably, cases where the CVO indicates that some operator did not achieve level, this allows us to actually look at the operand and see why the operator may not have achieved the level, or whether there is a disconnect between the operand's level and the operator's advertised level.